include/openthread/border_router.h

```diff
@@ -117,6 +117,21 @@ otError otBorderRoutingGetOmrPrefix(otInstance *aInstance, otIp6Prefix *aPrefix)
  */
 otError otBorderRoutingGetOnLinkPrefix(otInstance *aInstance, otIp6Prefix *aPrefix);
 
+/**
+ * This method returns the local NAT64 prefix, which is either a randomly
+ * generated 96-bit prefix or the Well-known Prefix 64:ff9b::/96.
+ *
+ * This prefix might not be advertised in the Thread network.
+ *
```

Comment:
Add a comment here that this API is only available under `OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE`?

Reply:
Updated